testsuite/pytests/test_connect_one_to_one.py

```diff
@@ -21,11 +21,10 @@
 
 import numpy as np
 import unittest
-import test_connect_helpers as hf
-from test_connect_parameters import TestParams
+import connect_test_base as hf
 
 
-class TestOneToOne(TestParams):
+class TestOneToOne(hf.ConnectTestBase):
```

Comment:
... and
```suggestion
class TestOneToOne(ConnectTestBase):
```
?

If you agree, this should be applied for all similar tests.

Reply:
Indeed!